src/services/persisters/s3Persister.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs');
+const AWS = require('aws-sdk');
+
+const bucketName = "mirai-app";
+const MongoDbPersister = require('./persisters/mongoDbPersister');
+
+
+class s3Persister {
```

Comment:
Capitalize S in `s3Persister` -> `S3Persister`. The capitalization convention we're going with for class names is pascal casing (`EveryFirstLetter`).